tests/scancode/data/long_file_name/0123456789012345678901234567890123456789.c

```diff
@@ -0,0 +1,1967 @@
+/* deflate.c -- compress data using the deflation algorithm
```

Comment:
Why use such a big file for tests when you only care about the name? and not the content? This is wasted space.